trace/trace_test.go

```diff
@@ -17,14 +17,17 @@ package trace
 import (
 	"context"
 	"fmt"
+	"go.opencensus.io/trace/tracestate"
 	"reflect"
 	"testing"
 	"time"
 )
 
 var (
-	tid = TraceID{1, 2, 3, 4, 5, 6, 7, 8, 1, 2, 4, 8, 16, 32, 64, 128}
-	sid = SpanID{1, 2, 4, 8, 16, 32, 64, 128}
+	tid               = TraceID{1, 2, 3, 4, 5, 6, 7, 8, 1, 2, 4, 8, 16, 32, 64, 128}
+	sid               = SpanID{1, 2, 4, 8, 16, 32, 64, 128}
+	defaultTracestate = tracestate.Tracestate{}
+	tracestate1, _    = tracestate.NewFromParent(nil, "foo", "bar")
```

Comment:
Want to give this a descriptive name? testTracestate?

Reply:
done.